### PR TITLE
client-cert caching interactions

### PR DESCRIPTION
**Pull request by b---c**
Attempt to cover client-cert and caching interactions per mnot's guidance in issue #1736

### CODE REVIEW

File: draft-ietf-httpbis-client-cert-field.md [hunk 1]
```diff
@@ -241,6 +241,15 @@ Forward proxies and other intermediaries MUST NOT add the `Client-Cert` or
 `Client-Cert` or `Client-Cert-Chain` header field. Similarly, clients MUST NOT
 employ the `Client-Cert` or `Client-Cert-Chain` header field in requests.
 
+When the value of the `Client-Cert` request header is used to select a response
+(e.g., the response content is access-controlled), the response MUST either be
+marked as uncacheable (e.g., `Cache-Control: no-store`) or be designated for
```
Comment (mnot) [inline draft-ietf-httpbis-client-cert-field.md:246]: It doesn't necessarily need to be _marked_  -- some responses just are uncacheable.

```suggestion
uncacheable (e.g., by sending `Cache-Control: no-store`) or be designated for
```

File: draft-ietf-httpbis-client-cert-field.md [hunk 2]
```diff
@@ -241,6 +241,15 @@ Forward proxies and other intermediaries MUST NOT add the `Client-Cert` or
 `Client-Cert` or `Client-Cert-Chain` header field. Similarly, clients MUST NOT
 employ the `Client-Cert` or `Client-Cert-Chain` header field in requests.
 
+When the value of the `Client-Cert` request header is used to select a response
```
Comment (mnot) [inline draft-ietf-httpbis-client-cert-field.md:244]:
```suggestion
When the value of the `Client-Cert` request header field is used to select a response
```

File: draft-ietf-httpbis-client-cert-field.md [hunk 3]
```diff
@@ -241,6 +241,15 @@ Forward proxies and other intermediaries MUST NOT add the `Client-Cert` or
...
+(e.g., the response content is access-controlled), the response MUST either be
+marked as uncacheable (e.g., `Cache-Control: no-store`) or be designated for
+selective reuse only for subsequent requests with the same `Client-Cert`
+header value (e.g., `Vary: client-cert`).
```
Comment (mnot) [inline draft-ietf-httpbis-client-cert-field.md:248]: `Vary` isn't just an example, it's the mechanism.

```suggestion
selective reuse only for subsequent requests with the same `Client-Cert`
header value by sending a `Vary: Client-Cert` response header.
```

**Review by mnot — CHANGES_REQUESTED** [review]
Looks pretty good, a couple of nits

**Comment by b---c** [comment]
Thanks Mark, accepted all your suggested changes

**Pull request merged**